packages/components/src/components/SquiggleViewer/ValueWithContextViewer.tsx

```diff
@@ -305,11 +325,39 @@ export const ValueWithContextViewer: FC<Props> = ({
               )}
               {!isOpen && <CommentIconForValue value={value} />}
             </div>
-            {enableDropdownMenu && (
-              <div className="inline-flex space-x-1 items-center">
-                <SquiggleValueMenu value={value} />
-              </div>
-            )}
+            <div className="inline-flex space-x-2 items-center">
+              {enableDropdownMenu && <SquiggleValueMenu value={value} />}
+              {exportData && exportData.path.length < 2 && onOpenExport && (
+                <TextTooltip
+                  text={
+                    `Go to model ${exportData.sourceId} ` +
+                    (!exportData.path.length
+                      ? "page"
+                      : "export " + exportData.path.join("/") + " page")
+                  }
+                  placement="bottom"
+                  offset={5}
+                >
+                  <div>
+                    <LinkIcon
+                      size={14}
+                      onClick={() =>
+                        onOpenExport(
+                          exportData.sourceId,
+                          exportData.path[0] || undefined
+                        )
```

Comment:
I'd like to cmd+click to open these in new tabs, but I can't.

This doesn't seem easily fixable with a single `onOpenExport`; simulating the precise behavior of native links is hard (also, links won't work properly before the hydration).

How about this?
- pass `linkComponent` through props and context (the appropriate type for it is `FC<PropsWithChildren<{ href: string }> | "a"`, defaulting to `"a"`)
- wrap `<LinkIcon>` with it (rename to capitalized version to use it in JSX)
- change `onOpenExport` to `getExportLink` or something

`linkComponent` could be in Playground props. Or, even better, we could have a single global provider for it in `@quri/ui`, because I suspect it'll be useful in a few other places too. (One example: `DropdownMenuLinkItem` vs `DropdownMenuNextLinkItem`).

It's not a problem if we improve this after this PR is merged, but I'd like to do it before the next release, to avoid versioned-components complications.

Reply:
Seems good to figure out in a different PR!